tests/qos/qos_sai_base.py

```diff
@@ -1576,3 +1576,44 @@ def resetWatermark(
         dut_asic.command("sleep 70")
         dut_asic.command("counterpoll watermark disable")
         dut_asic.command("counterpoll queue disable")
+
+    @pytest.fixture(scope='function', autouse=False)
+    def set_static_route(self, get_src_dst_asic_and_duts, dutTestParams, dutConfig):
+        # Get portchannels.
+        # find the one that is backplane based.
+        # set a static route through that portchannel.
+        # remove when done.
+        if dutTestParams["basicParams"]["sonic_asic_type"] != "cisco-8000":
+            yield
+            return
+        src_asic = get_src_dst_asic_and_duts['src_asic']
+        dst_asic = get_src_dst_asic_and_duts['dst_asic']
+        if src_asic == dst_asic:
+            yield
+            return
+        dst_ip = dutConfig["testPorts"]["dst_port_ip"]
```

Comment:
Some testcases use 3 dst ports, need static routes for dst_port_2_ip and dst_port_3_ip as well.

Could you add static routes to different next hops? Like this:
ip netns exec {src_asic} ip route add {dst_port_ip} via {bp_portchannel_ip}
ip netns exec {src_asic} ip route add {dst_port_2_ip} via {bp_portchannel_2_ip}
ip netns exec {src_asic} ip route add {dst_port_3_ip} via {bp_portchannel_3_ip}